src/main/java/edu/byu/ece/rapidSmith/interfaces/vivado/VivadoDesignCheckpoint.java

```diff
@@ -38,7 +38,7 @@
  * @author Thomas Townsend
  *
  */
-public final class TincrCheckpoint {
+public final class VivadoDesignCheckpoint {
```

Comment:
I think `VivadoCheckpoint` is a better name. There is more than just a `CellDesign` in this class. There is also a `Device `and `CellLibrary`.